test_tf2/test/buffer_core_test.cpp

```diff
@@ -1371,6 +1371,45 @@ TEST(BufferCore_lookupTransform, multi_configuration)
 // }
 //
 
+// Helpers used by gtest to print quaternions when errors happen
+// Must be defined in the namespace the class to be printed in lives, as per
+// https://github.com/google/googletest/blob/main/docs/advanced.md#teaching-googletest-how-to-print-your-values
+namespace geometry_msgs::msg
+{
+void PrintTo(const Quaternion & q, std::ostream * os)
```

Comment:
```suggestion
std::ostream& operator<<(std::ostream& stream, const geometry_msgs::msg::Quaternion & q)
```

Reply:
The PrintTo functions are not used....

Reply:
Check the comment, they are used on test failure